Library/Formula/mpw.rb

```diff
@@ -0,0 +1,32 @@
+require "formula"
+
+class Mpw < Formula
+  homepage "http://masterpasswordapp.com"
+  url "https://ssl.masterpasswordapp.com/mpw-2.1-cli3-3-gd5ff215.tar.gz"
```

Comment:
Can you add an explicit `version` tag? It's being misdetected as `215` due to the unusual filename.